specification/columns/consumedquantity.md

```diff
@@ -1,14 +1,15 @@
 # Consumed Quantity
 
-The Consumed Quantity represents the volume of a given SKU associated with a metered [*resource*](#glossary:resource) or [*service*](#glossary:service) used, based on the [Consumed Unit](#consumedunit). Consumed Quantity is often derived at a finer granularity or over a different time interval when compared to the [Pricing Quantity](#pricingquantity) (complementary to [Pricing Unit](#pricingunit)) and focuses on metered *resource* and *service* consumption, not pricing and cost.
+The Consumed Quantity represents the volume of a metered SKU associated with a [*resource*](#glossary:resource) or [*service*](#glossary:service) used, based on the [Consumed Unit](#consumedunit). Consumed Quantity is often derived at a finer granularity or over a different time interval when compared to the [Pricing Quantity](#pricingquantity) (complementary to [Pricing Unit](#pricingunit)) and focuses on *resource* and *service* consumption, not pricing and cost.
 
 The ConsumedQuantity column adheres to the following requirements:
 
 * ConsumedQuantity MUST be present in a FOCUS dataset when the provider supports the measurement of usage.
 * ConsumedQuantity MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements.
-* ConsumedQuantity MUST NOT be null if [ChargeCategory](#chargecategory) is "Usage" and [ChargeClass](#chargeclass) is not "Correction".
-* ConsumedQuantity MUST be null if [CommitmentDiscountStatus](#commitmentdiscountstatus) is "Unused" or for other *ChargeCategory* values.
-* ConsumedQuantity MAY be negative or null in cases where *ChargeClass* is "Correction".
+* If [ChargeCategory](#chargecategory) is "Usage" and [CommitmentDiscountStatus](#commitmentdiscountstatus) is "Used", the following applies:
```

Comment:
How about this more simplified version?

```
* ConsumedQuantity MUST be null if [ChargeCategory](#chargecategory) is "Usage" and [CommitmentDiscountStatus](#commitmentdiscountstatus) is "Unused".
* ConsumedQuantity MAY be negative or null in cases where [ChargeClass](#chargeclass) is "Correction".
* ConsumedQuantity MUST NOT be null in all other cases.
```

Reply:
There are some bugs in that suggestion, but I’ll follow the suggested principle and provide a new version.

Reply:
Chris' suggestion:
* ConsumedQuantity MUST be null unless [*ChargeCategory*](#chargecategory) is "Usage" and [CommitmentDiscountStatus](#commitmentdiscountstatus) is not "Unused".
* ConsumedQuantity MAY be negative or null in cases where [ChargeClass](#chargeclass) is "Correction".